test/i18n.py

```diff
@@ -1,11 +1,11 @@
 from unittest import TestCase
-from tcii18n import trans
+from tcii18n.translator import Translator
 
 
 class I18nTest(TestCase):
     def setUp(self):
-        self.trans = trans._
-        self.trans_it = None
+        self.trans = Translator('test/fixtures.fr.csv').translate
```

Comment:
for avoiding multiple instance calls as you mentioned in the previous comment, you probably want to instanciate only once:
```
translator = Translator('test/fixtures.fr.csv')
self.trans = translator.translate
```
and similarly for `self.trans_it`